app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
1. You can write `"somekey" in somedict` to check if the dictionary contains such key.
2. `flake8` wants you to write `vstr["vaccine"] is None` instead of `vstr["vaccine"] == None`.
3. You [can use](https://docs.python.org/3/library/stdtypes.html#dict.get) `somedict.get("somekey")` to simplify:

```suggestion
        if vstr.get("vaccine") is None:
```

, or

```suggestion
        if not vstr.get("vaccine"):
```